lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -507,13 +507,12 @@ defmodule Kernel.SpecialForms do
   defmacro alias(module, opts), do: error!([module, opts])
 
   @doc """
-  Requires a given module to be compiled and loaded.
+  Tells Elixir you want to use the macros defined in a module.
```

Comment:
@whatyouhide didn't see that last remark before writing my last comment. Good then, I'll revise the patch in a minute.